### ISSUE TITLE
ICE: fully private item in the table

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang/rust#104518]
### Code
needs debug assertions!
```Rust
#![crate_type = "rlib"]
#![cfg_attr(not(feature = "cargo-build"), feature(staged_api, core, no_std))]

pub mod sysconf {}

pub mod sysconf {
    pub static _SC_NPROCESSORS_ONLN: c_int = 84;
}
```

### Meta

`rustc --version --verbose`:
```
e69336efe035c274f2ee66922cd9cac6015956ea
```

### Error output

```
error[E0428]: the name `sysconf` is defined multiple times
 --> 310b743961389e3451c10cc3fb8a2104bbaae065.rs:6:1
  |
4 | pub mod sysconf {}
  | --------------- previous definition of the module `sysconf` here
5 |
6 | pub mod sysconf {
  | ^^^^^^^^^^^^^^^ `sysconf` redefined here
  |
  = note: `sysconf` must be defined only once in the type namespace of this module

error[E0412]: cannot find type `c_int` in this scope
 --> 310b743961389e3451c10cc3fb8a2104bbaae065.rs:7:38
  |
7 |     pub static _SC_NPROCESSORS_ONLN: c_int = 84;
  |                                      ^^^^^ not found in this scope
  |
help: consider importing one of these items
  |
7 |     use std::ffi::c_int;
  |
7 |     use std::os::raw::c_int;
  |
```

<details><summary><strong>Backtrace</strong></summary>
<p>

```
error: internal compiler error: compiler/rustc_middle/src/middle/privacy.rs:203:17: fully private item in the table DefId(0:5 ~ 310b743961389e3451c10cc3fb8a2104bbaae065[92fb]::sysconf#1::_SC_NPROCESSORS_ONLN): Restricted(DefId(0:4 ~ 310b743961389e3451c10cc3fb8a2104bbaae065[92fb]::sysconf#1))
 --> 310b743961389e3451c10cc3fb8a2104bbaae065.rs:7:5
  |
7 |     pub static _SC_NPROCESSORS_ONLN: c_int = 84;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

thread 'rustc' panicked at 'Box<dyn Any>', /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_errors/src/lib.rs:967:33
stack backtrace:
   0:     0x7fa485ced23e - std::backtrace_rs::backtrace::libunwind::trace::h2ba7924f373abcbe
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/../../backtrace/src/backtrace/libunwind.rs:93:5
   1:     0x7fa485ced23e - std::backtrace_rs::backtrace::trace_unsynchronized::h5a4fdc5ca15c5a11
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x7fa485ced23e - std::sys_common::backtrace::_print_fmt::h34d240f86b681b56
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/sys_common/backtrace.rs:65:5
   3:     0x7fa485ced23e - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hff0170f2c82d7337
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/sys_common/backtrace.rs:44:22
   4:     0x7fa485d6bc08 - core::fmt::write::h1efe05f8990e10dd
                               at /home/matthias/vcs/github/rust_debug_assertions/library/core/src/fmt/mod.rs:1208:17
   5:     0x7fa485cbd0af - std::io::Write::write_fmt::h09b98dcf13d06d7e
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/io/mod.rs:1682:15
   6:     0x7fa485ced035 - std::sys_common::backtrace::_print::hd796cba033e68e9f
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/sys_common/backtrace.rs:47:5
   7:     0x7fa485ced035 - std::sys_common::backtrace::print::h372132b3c3624c25
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/sys_common/backtrace.rs:34:9
   8:     0x7fa485cc9be7 - std::panicking::default_hook::{{closure}}::h25ec0b66e0c6b3e1
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:267:22
   9:     0x7fa485cc9980 - std::panicking::default_hook::h57d3b2570aeea246
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:286:9
  10:     0x7fa486b147d2 - <alloc[19a8b8a34660badf]::boxed::Box<dyn for<'a, 'b> core[54e86a843f8c5207]::ops::function::Fn<(&'a core[54e86a843f8c5207]::panic::panic_info::PanicInfo<'b>,), Output = ()> + core[54e86a843f8c5207]::marker::Sync + core[54e86a843f8c5207]::marker::Send> as core[54e86a843f8c5207]::ops::function::Fn<(&core[54e86a843f8c5207]::panic::panic_info::PanicInfo,)>>::call
                               at /home/matthias/vcs/github/rust_debug_assertions/library/alloc/src/boxed.rs:2024:9
  11:     0x7fa486b147d2 - rustc_driver[102e08cae241c12c]::DEFAULT_HOOK::{closure#0}::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_driver/src/lib.rs:1207:13
  12:     0x7fa485cca5cc - <alloc::boxed::Box<F,A> as core::ops::function::Fn<Args>>::call::he864250e2a0af0c3
                               at /home/matthias/vcs/github/rust_debug_assertions/library/alloc/src/boxed.rs:2024:9
  13:     0x7fa485cca5cc - std::panicking::rust_panic_with_hook::h57f8ea81dc24fd24
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:692:13
  14:     0x7fa48bfe2093 - std[b60d3abfffc60fec]::panicking::begin_panic::<rustc_errors[6b61d4e931019684]::ExplicitBug>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:608:9
  15:     0x7fa48bfdac26 - std[b60d3abfffc60fec]::sys_common::backtrace::__rust_end_short_backtrace::<std[b60d3abfffc60fec]::panicking::begin_panic<rustc_errors[6b61d4e931019684]::ExplicitBug>::{closure#0}, !>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/sys_common/backtrace.rs:137:18
  16:     0x7fa486a84d46 - std[b60d3abfffc60fec]::panicking::begin_panic::<rustc_errors[6b61d4e931019684]::ExplicitBug>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:607:12
  17:     0x7fa48bfdac16 - std[b60d3abfffc60fec]::panic::panic_any::<rustc_errors[6b61d4e931019684]::ExplicitBug>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panic.rs:61:5
  18:     0x7fa48bfd10e1 - <rustc_errors[6b61d4e931019684]::HandlerInner>::span_bug::<rustc_span[18dcca4f631a4712]::span_encoding::Span, &alloc[19a8b8a34660badf]::string::String>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_errors/src/lib.rs:1493:9
  19:     0x7fa48bfd0e87 - <rustc_errors[6b61d4e931019684]::Handler>::span_bug::<rustc_span[18dcca4f631a4712]::span_encoding::Span, &alloc[19a8b8a34660badf]::string::String>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_errors/src/lib.rs:967:9
  20:     0x7fa48c182904 - rustc_middle[137db65a67c432c]::util::bug::opt_span_bug_fmt::<rustc_span[18dcca4f631a4712]::span_encoding::Span>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/util/bug.rs:34:40
  21:     0x7fa48c182904 - rustc_middle[137db65a67c432c]::ty::context::tls::with_opt::<rustc_middle[137db65a67c432c]::util::bug::opt_span_bug_fmt<rustc_span[18dcca4f631a4712]::span_encoding::Span>::{closure#0}, ()>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1994:40
  22:     0x7fa48c182904 - rustc_middle[137db65a67c432c]::ty::context::tls::with_context_opt::<rustc_middle[137db65a67c432c]::ty::context::tls::with_opt<rustc_middle[137db65a67c432c]::util::bug::opt_span_bug_fmt<rustc_span[18dcca4f631a4712]::span_encoding::Span>::{closure#0}, ()>::{closure#0}, ()>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1946:22
  23:     0x7fa48c183e27 - rustc_middle[137db65a67c432c]::ty::context::tls::with_opt::<rustc_middle[137db65a67c432c]::util::bug::opt_span_bug_fmt<rustc_span[18dcca4f631a4712]::span_encoding::Span>::{closure#0}, ()>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1994:9
  24:     0x7fa48c183e27 - rustc_middle[137db65a67c432c]::util::bug::opt_span_bug_fmt::<rustc_span[18dcca4f631a4712]::span_encoding::Span>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/util/bug.rs:31:5
  25:     0x7fa486a8a6d5 - rustc_middle[137db65a67c432c]::util::bug::span_bug_fmt::<rustc_span[18dcca4f631a4712]::span_encoding::Span>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/util/bug.rs:22:5
  26:     0x7fa48c1a9931 - <rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::check_invariants
  27:     0x7fa4895ffdc0 - rustc_privacy[7237ed65e3f863c3]::effective_visibilities
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_privacy/src/lib.rs:2142:5
  28:     0x7fa48afb1071 - <rustc_query_system[81dab959ca583418]::query::config::QueryVTable<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>>::compute
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/config.rs:65:9
  29:     0x7fa48afb1071 - rustc_query_system[81dab959ca583418]::query::plumbing::execute_job::<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:418:13
  30:     0x7fa48afb1071 - stacker[67c4101402540900]::maybe_grow::<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>
                               at /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/stacker-0.1.15/src/lib.rs:55:9
  31:     0x7fa48afb1071 - rustc_data_structures[2dc78f4d764e2a91]::stack::ensure_sufficient_stack::<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_data_structures/src/stack.rs:17:5
  32:     0x7fa48afb1071 - <rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query::<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:128:17
  33:     0x7fa48afb1071 - rustc_middle[137db65a67c432c]::ty::context::tls::enter_context::<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1929:50
  34:     0x7fa48afb1071 - rustc_middle[137db65a67c432c]::ty::context::tls::set_tlv::<rustc_middle[137db65a67c432c]::ty::context::tls::enter_context<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1913:9
  35:     0x7fa48afb1071 - rustc_middle[137db65a67c432c]::ty::context::tls::enter_context::<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1929:9
  36:     0x7fa48afb1071 - <rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query::<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:127:13
  37:     0x7fa48afb1071 - rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context::<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1973:13
  38:     0x7fa48afb1071 - rustc_middle[137db65a67c432c]::ty::context::tls::with_context::<rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1957:40
  39:     0x7fa48afb1071 - rustc_middle[137db65a67c432c]::ty::context::tls::with_context_opt::<rustc_middle[137db65a67c432c]::ty::context::tls::with_context<rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1946:22
  40:     0x7fa48afb1071 - rustc_middle[137db65a67c432c]::ty::context::tls::with_context::<rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1957:9
  41:     0x7fa48afb1071 - rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context::<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>::{closure#0}, &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1970:9
  42:     0x7fa48afb1071 - <rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query::<&rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>::{closure#0}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:112:9
  43:     0x7fa48afb1071 - rustc_query_system[81dab959ca583418]::query::plumbing::execute_job::<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:417:22
  44:     0x7fa48afb1071 - rustc_query_system[81dab959ca583418]::query::plumbing::try_execute_query::<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, rustc_query_system[81dab959ca583418]::query::caches::DefaultCache<(), &rustc_middle[137db65a67c432c]::middle::privacy::EffectiveVisibilities>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:376:44
  45:     0x7fa48b050cf6 - rustc_query_system[81dab959ca583418]::query::plumbing::get_query::<rustc_query_impl[e3b674dd72cd8280]::queries::effective_visibilities, rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:740:36
  46:     0x7fa48ab89e0a - <rustc_query_impl[e3b674dd72cd8280]::Queries as rustc_middle[137db65a67c432c]::ty::query::QueryEngine>::effective_visibilities::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:708:17
  47:     0x7fa48ab89e0a - <rustc_query_impl[e3b674dd72cd8280]::Queries as rustc_middle[137db65a67c432c]::ty::query::QueryEngine>::effective_visibilities
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/lib.rs:47:1
  48:     0x7fa48a014921 - <rustc_middle[137db65a67c432c]::ty::query::TyCtxtAt>::effective_visibilities
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/query.rs:261:17
  49:     0x7fa48a014921 - <rustc_middle[137db65a67c432c]::ty::context::TyCtxt>::effective_visibilities
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/query.rs:242:17
  50:     0x7fa48a014921 - rustc_passes[36204dc4d3cae20]::stability::check_unused_or_stable_features
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_passes/src/stability.rs:933:39
  51:     0x7fa486c91be5 - rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}::{closure#2}::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/passes.rs:859:21
  52:     0x7fa486c91be5 - <rustc_data_structures[2dc78f4d764e2a91]::profiling::VerboseTimingGuard>::run::<(), rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}::{closure#2}::{closure#0}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_data_structures/src/profiling.rs:727:9
  53:     0x7fa486c91be5 - <rustc_session[ee121006aeecfdd7]::session::Session>::time::<(), rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}::{closure#2}::{closure#0}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_session/src/utils.rs:10:9
  54:     0x7fa486c91be5 - rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}::{closure#2}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/passes.rs:858:17
  55:     0x7fa486c91be5 - <rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}::{closure#2} as core[54e86a843f8c5207]::ops::function::FnOnce<()>>::call_once
                               at /home/matthias/vcs/github/rust_debug_assertions/library/core/src/ops/function.rs:422:5
  56:     0x7fa486c91be5 - <core[54e86a843f8c5207]::panic::unwind_safe::AssertUnwindSafe<rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}::{closure#2}> as core[54e86a843f8c5207]::ops::function::FnOnce<()>>::call_once
                               at /home/matthias/vcs/github/rust_debug_assertions/library/core/src/panic/unwind_safe.rs:271:9
  57:     0x7fa486c91be5 - std[b60d3abfffc60fec]::panicking::try::do_call::<core[54e86a843f8c5207]::panic::unwind_safe::AssertUnwindSafe<rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}::{closure#2}>, ()>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:483:40
  58:     0x7fa486c91be5 - std[b60d3abfffc60fec]::panicking::try::<(), core[54e86a843f8c5207]::panic::unwind_safe::AssertUnwindSafe<rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}::{closure#2}>>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:447:19
  59:     0x7fa486c91be5 - std[b60d3abfffc60fec]::panic::catch_unwind::<core[54e86a843f8c5207]::panic::unwind_safe::AssertUnwindSafe<rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}::{closure#2}>, ()>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panic.rs:137:14
  60:     0x7fa486c91be5 - rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/passes.rs:838:9
  61:     0x7fa486c91be5 - <rustc_data_structures[2dc78f4d764e2a91]::profiling::VerboseTimingGuard>::run::<(), rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_data_structures/src/profiling.rs:727:9
  62:     0x7fa486c91be5 - <rustc_session[ee121006aeecfdd7]::session::Session>::time::<(), rustc_interface[504271c6e2da38b2]::passes::analysis::{closure#0}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_session/src/utils.rs:10:9
  63:     0x7fa486cdd099 - rustc_interface[504271c6e2da38b2]::passes::analysis
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/passes.rs:837:5
  64:     0x7fa48afa6ee9 - <rustc_query_system[81dab959ca583418]::query::config::QueryVTable<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>>::compute
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/config.rs:65:9
  65:     0x7fa48afa6ee9 - rustc_query_system[81dab959ca583418]::query::plumbing::execute_job::<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:418:13
  66:     0x7fa48afa6ee9 - stacker[67c4101402540900]::maybe_grow::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>
                               at /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/stacker-0.1.15/src/lib.rs:55:9
  67:     0x7fa48afa6ee9 - rustc_data_structures[2dc78f4d764e2a91]::stack::ensure_sufficient_stack::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_data_structures/src/stack.rs:17:5
  68:     0x7fa48afa6ee9 - <rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:128:17
  69:     0x7fa48afa6ee9 - rustc_middle[137db65a67c432c]::ty::context::tls::enter_context::<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1929:50
  70:     0x7fa48afa6ee9 - rustc_middle[137db65a67c432c]::ty::context::tls::set_tlv::<rustc_middle[137db65a67c432c]::ty::context::tls::enter_context<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1913:9
  71:     0x7fa48afa6ee9 - rustc_middle[137db65a67c432c]::ty::context::tls::enter_context::<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1929:9
  72:     0x7fa48afa6ee9 - <rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:127:13
  73:     0x7fa48afa6ee9 - rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context::<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1973:13
  74:     0x7fa48afa6ee9 - rustc_middle[137db65a67c432c]::ty::context::tls::with_context::<rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1957:40
  75:     0x7fa48afa6ee9 - rustc_middle[137db65a67c432c]::ty::context::tls::with_context_opt::<rustc_middle[137db65a67c432c]::ty::context::tls::with_context<rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1946:22
  76:     0x7fa48afa6ee9 - rustc_middle[137db65a67c432c]::ty::context::tls::with_context::<rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1957:9
  77:     0x7fa48afa6ee9 - rustc_middle[137db65a67c432c]::ty::context::tls::with_related_context::<<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1970:9
  78:     0x7fa48afa6ee9 - <rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt as rustc_query_system[81dab959ca583418]::query::QueryContext>::start_query::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_query_system[81dab959ca583418]::query::plumbing::execute_job<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:112:9
  79:     0x7fa48afa6ee9 - rustc_query_system[81dab959ca583418]::query::plumbing::execute_job::<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, (), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:417:22
  80:     0x7fa48afa6ee9 - rustc_query_system[81dab959ca583418]::query::plumbing::try_execute_query::<rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt, rustc_query_system[81dab959ca583418]::query::caches::DefaultCache<(), core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:376:44
  81:     0x7fa48b059696 - rustc_query_system[81dab959ca583418]::query::plumbing::get_query::<rustc_query_impl[e3b674dd72cd8280]::queries::analysis, rustc_query_impl[e3b674dd72cd8280]::plumbing::QueryCtxt>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_system/src/query/plumbing.rs:740:36
  82:     0x7fa48ab5d61a - <rustc_query_impl[e3b674dd72cd8280]::Queries as rustc_middle[137db65a67c432c]::ty::query::QueryEngine>::analysis::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/plumbing.rs:708:17
  83:     0x7fa48ab5d61a - <rustc_query_impl[e3b674dd72cd8280]::Queries as rustc_middle[137db65a67c432c]::ty::query::QueryEngine>::analysis
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_query_impl/src/lib.rs:47:1
  84:     0x7fa486b8f217 - <rustc_middle[137db65a67c432c]::ty::query::TyCtxtAt>::analysis
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/query.rs:261:17
  85:     0x7fa486b8f217 - <rustc_middle[137db65a67c432c]::ty::context::TyCtxt>::analysis
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/query.rs:242:17
  86:     0x7fa486b8f217 - rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}::{closure#2}::{closure#3}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_driver/src/lib.rs:377:30
  87:     0x7fa486b8f217 - <rustc_interface[504271c6e2da38b2]::passes::QueryContext>::enter::<rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}::{closure#2}::{closure#3}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/passes.rs:765:42
  88:     0x7fa486b8f217 - rustc_middle[137db65a67c432c]::ty::context::tls::enter_context::<<rustc_interface[504271c6e2da38b2]::passes::QueryContext>::enter<rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}::{closure#2}::{closure#3}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1929:50
  89:     0x7fa486b8f217 - rustc_middle[137db65a67c432c]::ty::context::tls::set_tlv::<rustc_middle[137db65a67c432c]::ty::context::tls::enter_context<<rustc_interface[504271c6e2da38b2]::passes::QueryContext>::enter<rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}::{closure#2}::{closure#3}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1913:9
  90:     0x7fa486b8f217 - rustc_middle[137db65a67c432c]::ty::context::tls::enter_context::<<rustc_interface[504271c6e2da38b2]::passes::QueryContext>::enter<rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}::{closure#2}::{closure#3}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_middle/src/ty/context.rs:1929:9
  91:     0x7fa486b8f217 - <rustc_interface[504271c6e2da38b2]::passes::QueryContext>::enter::<rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}::{closure#2}::{closure#3}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/passes.rs:765:9
  92:     0x7fa486bac3d5 - rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}::{closure#2}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_driver/src/lib.rs:376:13
  93:     0x7fa486bac3d5 - <rustc_interface[504271c6e2da38b2]::interface::Compiler>::enter::<rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}::{closure#2}, core[54e86a843f8c5207]::result::Result<core[54e86a843f8c5207]::option::Option<rustc_interface[504271c6e2da38b2]::queries::Linker>, rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/queries.rs:381:19
  94:     0x7fa486b16335 - rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_driver/src/lib.rs:308:22
  95:     0x7fa486b16335 - rustc_interface[504271c6e2da38b2]::interface::run_compiler::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}::{closure#1}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/interface.rs:327:21
  96:     0x7fa486b16335 - rustc_span[18dcca4f631a4712]::with_source_map::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}::{closure#1}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_span/src/lib.rs:1009:5
  97:     0x7fa486b9a92c - rustc_interface[504271c6e2da38b2]::interface::run_compiler::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/interface.rs:321:13
  98:     0x7fa486b9a92c - <scoped_tls[3f7a8f801319db86]::ScopedKey<rustc_span[18dcca4f631a4712]::SessionGlobals>>::set::<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/scoped-tls-1.0.0/src/lib.rs:137:9
  99:     0x7fa486b95ec0 - rustc_span[18dcca4f631a4712]::create_session_globals_then::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}>
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_span/src/lib.rs:111:5
 100:     0x7fa486b95ec0 - rustc_interface[504271c6e2da38b2]::util::run_in_thread_pool_with_globals::<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/compiler/rustc_interface/src/util.rs:148:38
 101:     0x7fa486b95ec0 - std[b60d3abfffc60fec]::sys_common::backtrace::__rust_begin_short_backtrace::<rustc_interface[504271c6e2da38b2]::util::run_in_thread_pool_with_globals<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/sys_common/backtrace.rs:121:18
 102:     0x7fa486b2a769 - <std[b60d3abfffc60fec]::thread::Builder>::spawn_unchecked_::<rustc_interface[504271c6e2da38b2]::util::run_in_thread_pool_with_globals<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#1}::{closure#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/thread/mod.rs:551:17
 103:     0x7fa486b2a769 - <core[54e86a843f8c5207]::panic::unwind_safe::AssertUnwindSafe<<std[b60d3abfffc60fec]::thread::Builder>::spawn_unchecked_<rustc_interface[504271c6e2da38b2]::util::run_in_thread_pool_with_globals<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#1}::{closure#0}> as core[54e86a843f8c5207]::ops::function::FnOnce<()>>::call_once
                               at /home/matthias/vcs/github/rust_debug_assertions/library/core/src/panic/unwind_safe.rs:271:9
 104:     0x7fa486b2a769 - std[b60d3abfffc60fec]::panicking::try::do_call::<core[54e86a843f8c5207]::panic::unwind_safe::AssertUnwindSafe<<std[b60d3abfffc60fec]::thread::Builder>::spawn_unchecked_<rustc_interface[504271c6e2da38b2]::util::run_in_thread_pool_with_globals<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#1}::{closure#0}>, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:483:40
 105:     0x7fa486b2a769 - std[b60d3abfffc60fec]::panicking::try::<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, core[54e86a843f8c5207]::panic::unwind_safe::AssertUnwindSafe<<std[b60d3abfffc60fec]::thread::Builder>::spawn_unchecked_<rustc_interface[504271c6e2da38b2]::util::run_in_thread_pool_with_globals<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#1}::{closure#0}>>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panicking.rs:447:19
 106:     0x7fa486b2a769 - std[b60d3abfffc60fec]::panic::catch_unwind::<core[54e86a843f8c5207]::panic::unwind_safe::AssertUnwindSafe<<std[b60d3abfffc60fec]::thread::Builder>::spawn_unchecked_<rustc_interface[504271c6e2da38b2]::util::run_in_thread_pool_with_globals<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#1}::{closure#0}>, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/panic.rs:137:14
 107:     0x7fa486b2a769 - <std[b60d3abfffc60fec]::thread::Builder>::spawn_unchecked_::<rustc_interface[504271c6e2da38b2]::util::run_in_thread_pool_with_globals<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#1}
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/thread/mod.rs:550:30
 108:     0x7fa486b2a769 - <<std[b60d3abfffc60fec]::thread::Builder>::spawn_unchecked_<rustc_interface[504271c6e2da38b2]::util::run_in_thread_pool_with_globals<rustc_interface[504271c6e2da38b2]::interface::run_compiler<core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>, rustc_driver[102e08cae241c12c]::run_compiler::{closure#1}>::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#0}::{closure#0}, core[54e86a843f8c5207]::result::Result<(), rustc_errors[6b61d4e931019684]::ErrorGuaranteed>>::{closure#1} as core[54e86a843f8c5207]::ops::function::FnOnce<()>>::call_once::{shim:vtable#0}
                               at /home/matthias/vcs/github/rust_debug_assertions/library/core/src/ops/function.rs:422:5
 109:     0x7fa485ca7058 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::hf434786bd508e033
                               at /home/matthias/vcs/github/rust_debug_assertions/library/alloc/src/boxed.rs:1990:9
 110:     0x7fa485ca7058 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::h79e82a56a42c42f5
                               at /home/matthias/vcs/github/rust_debug_assertions/library/alloc/src/boxed.rs:1990:9
 111:     0x7fa485c8f895 - std::sys::unix::thread::Thread::new::thread_start::he8a1fd9beacdaef9
                               at /home/matthias/vcs/github/rust_debug_assertions/library/std/src/sys/unix/thread.rs:108:17
 112:     0x7fa485a9f8fd - <unknown>
 113:     0x7fa485b21a60 - <unknown>
 114:                0x0 - <unknown>

note: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust/issues/new?labels=C-bug%2C+I-ICE%2C+T-compiler&template=ice.md

note: rustc 1.67.0-dev running on x86_64-unknown-linux-gnu

query stack during panic:
#0 [effective_visibilities] checking effective visibilities
#1 [analysis] running analysis passes on this crate
end of query stack
error: aborting due to 3 previous errors

Some errors have detailed explanations: E0412, E0428.
For more information about an error, try `rustc --explain E0412`.
```

</p>
</details>

**Comment by petrochenkov** [comment]
This may be a duplicate of https://github.com/rust-lang/rust/issues/104249.
